src/screens/Home/index.js

```diff
@@ -31,10 +49,24 @@ export default ({navigation}) => {
   const onButtonPress = () => {
     navigation.navigate('Book')
   }
-
+ 
   return (
     <View style={styles.container}>
       <Header/>
+      <FlatList
+        key={numColumns}
```

Comment:
mau nanya ini key tuh apa? (nanya aja) jawab di comment aja

Reply:
untuk paksa render ulang komponen. Kalo numColumns doang yang diubah eror "changing numColumns on the fly is not supported"